lib/travis/api/v3/services/cron/for_branch.rb

```diff
@@ -0,0 +1,8 @@
+module Travis::API::V3
+  class Services::Cron::ForBranch < Service
+
+    def run!
+      Models::Cron.where(:branch_id => find(:branch, find(:repository))).first
```

Comment:
I'd love if you could move this into a query method. We try to encapsulate anything AR specific there.